### DISCUSSION TITLE
Hw3

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#166] [Q&A]
npz_coords = np.load('projectile_coords.npz')
t = npz_coords['t']
x = npz_coords['x']
y = npz_coords['y']

I have copy/paste this but I keep getting an error like this: 

 [Errno 2] No such file or directory: 'projectile_coords.npz'

What am I doing wrong?

**Comment by AmanuelDagnew** [comment]
Try this,
npz_coords = np.load('../data/projectile_coords.npz')
